src/modules/elements/action-dashboard/style.pcss

```diff
@@ -1,67 +1,69 @@
-.tribe-editor__action-dashboard {
-	background-color: #f8f9fb;
-	display: flex;
-	position: relative;
+.tribe-editor__tickets {
+	.tribe-editor__action-dashboard {
+		background-color: #f8f9fb;
```

Comment:
(old) non-palette color?

Reply:
Applied the closest color from the palette now, btw that color is from 5 years ago according to git  😄  I've just applied the alphabetical order